android/app/src/main/res/drawable/ic_reply_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Not needed

Reply:
Probably a trash from another branch... removing...